core/toolbox.js

```diff
@@ -427,8 +427,7 @@ Blockly.Toolbox.prototype.clearSelection = function() {
  * @package
  */
 Blockly.Toolbox.prototype.addStyle = function(style) {
-  Blockly.utils.addClass(/** @type {!Element} */ (this.HtmlDiv),
-                         style);
+  Blockly.utils.addClass(/** @type {!Element} */ (this.HtmlDiv), style);
```

Comment:
`Toolbox.HtmlDiv` is only undefined prior to `init()`, but I don't see any reason why the constructor couldn't assign this earlier (calling `createDom()` earlier) to make this `@type {!Element}` in all cases, eliminating all of these casts.

If we don't do that, are the parenthesis around this.HtmlDiv necessary for the type annotation?

And finally, unrelatedly, why is the 'H' uppercase in the Toolbox field?

Reply:
Not handling in this PR.